tests/func/test_get.py

```diff
@@ -36,6 +36,31 @@ def test_get_repo_dir(repo_dir, erepo):
     trees_equal(src, dst)
 
 
+def test_get_regular_file(erepo):
+    src = os.path.join(erepo.root_dir, "file")
+    dst = src + "_imported"
+    erepo.create(src, "hello")
```

Comment:
How does it work if we haven't even added this new file to git? We are probably not testing it hard enough or smth.